internal/command/info/info.go

```diff
@@ -14,6 +14,7 @@ import (
 	"github.com/spf13/cobra"
 )
 
+// TODO: Deprecate and remove
```

Comment:
This line has multiplied throughout the repo, but I don't think we need it, since you just did deprecate it and we may not want to remove it.

Reply:
Removed it